src/Discord.Net.Interactions/Attributes/Commands/ModalInteractionAttribute.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+
+namespace Discord.Interactions
+{
+    /// <summary>
+    ///     Create a Modal interaction handler, CustomId represents
+    ///     the CustomId of the Modal that will be handled.
+    /// </summary>
+    /// <remarks>
+    ///     <see cref="GroupAttribute"/>s will add prefixes to this command if <see cref="IgnoreGroupNames"/> is set to <see langword="false"/>
+    ///     CustomID supports a Wild Card pattern where you can use the <see cref="InteractionServiceConfig.WildCardExpression"/> to match a set of CustomIDs.
```

Comment:
> CustomID

Is there going to be consistency between the naming of this property? (i.e. `CustomID` vs `CustomId`)